tests/acceptance/features/webUIFiles/copy.feature

```diff
@@ -64,3 +64,17 @@ Feature: copy files and folders
     And file "data.zip" should be listed on the webUI
     And as "user1" file "simple-folder/simple-empty-folder/data.zip" should exist
     And as "user1" file "simple-folder/data.zip" should exist
+
+    Scenario: copy a file into another folder with no change permission
+      Given user "user2" has been created with default attributes
+      And user "user2" has shared folder "simple-folder" with user "user1" with "read" permissions
+      And user "user1" has logged in using the webUI
+      When the user tries to copy file "lorem.txt" into folder "simple-folder (2)" using the webUI
+      Then it should not be possible to navigate into folder "simple-folder (2)" using the webUI
+
+  Scenario: copy a folder into another folder with no change permission
+    Given user "user2" has been created with default attributes
+    And user "user2" has shared folder "simple-folder" with user "user1" with "read" permissions
+    And user "user1" has logged in using the webUI
+    When the user tries to copy folder "simple-empty-folder" into folder "simple-folder (2)" using the webUI
+    Then it should not be possible to navigate into folder "simple-folder (2)" using the webUI
```

Comment:
I'm confused by this step, it should not be possible to copy into a read-only folder, but it should be possible to access the folder after the invalid try. Is that a bug?

Reply:
it should not be possible to copy into a read-only folder is demonstrated in this step mentioning that the folder is disabled